app/src/main/java/service/ForecastService.kt

```diff
@@ -71,11 +72,13 @@ class ForecastService(application: Application) {
         }
     }
 
-    fun getLocalRetrofitWeatherDAO() : IWeatherDAO {
-    TODO("Not yet implemented")
-        val db = Room.databaseBuilder(application, MapsActivityCurrentPlace::class.java, "DroneRadar").build()
+    fun getLocalRetrofitWeatherDAO(): IWeatherDAO {
+        TODO("Not yet implemented")
+/*        val db =
```

Comment:
commented this out as MapsActivityCurrentPlace is not a room database so was providing errors